server/RUNNING.md

```diff
@@ -43,10 +43,10 @@ Or interact with the other services in the cluster, e.g. connect to the DB
 
 Or interact with the MinIO S3 API
 
-    AWS_ACCESS_KEY_ID=test AWS_SECRET_ACCESS_KEY=testtest \
-        aws s3 --endpoint-url http://localhost:3200 ls s3://test
+    AWS_ACCESS_KEY_ID=test AWS_SECRET_ACCESS_KEY=changeme1234 \
```

Comment:
`AWS_ACCESS_KEY_ID` needs changing. Also the original command seems to have been wrong, the part after s3:// should be the bucket name, eg.
```
AWS_ACCESS_KEY_ID=changeme AWS_SECRET_ACCESS_KEY=changeme1234 \
    aws s3 --endpoint-url http://localhost:3200 ls s3://b2-eu-cen
```